Add http connector

WiiPlayer2 opened the pull request:
Adds `HttpConnector` with trigger `webhook` and action `request`.

WiiPlayer2 committed:
Add HttpConnector

WiiPlayer2 committed:
Add http.request

itshoro reviewed (dismissed):
Seems good, we should merge the dev branch first though.

itshoro committed:
Merge remote-tracking branch 'origin/dev' into feature/http-connector

itshoro committed:
Re-Add Connectors to ConnectorRegistry

I accidentally removed the connectors during the merge, so I'm re-adding them here.
